be/src/util/thread.cpp

```diff
@@ -74,18 +78,17 @@ class ThreadMgr {
     // already been removed, this is a no-op.
     void remove_thread(const pthread_t& pthread_id, const std::string& category);
 
-private:
+    void start_instrumentation(const WebPageHandler::ArgumentMap& args, EasyJson* ej) const;
```

Comment:
I think we'd better call this function a xxx_handler, it's a callback of a http request.

Reply:
done